api/provider/aws/test_aws/helpers.go

```diff
@@ -52,3 +55,55 @@ func deleteSGHelper(mockAWS *awsc.MockClient, securityGroupID string) {
 		DeleteSecurityGroup(input).
 		Return(&ec2.DeleteSecurityGroupOutput{}, nil)
 }
+
+func healthCheckHelper(healthCheck *models.HealthCheck) *elb.HealthCheck {
+	if healthCheck == nil {
+		elbHealthCheck := &elb.HealthCheck{}
+		elbHealthCheck.SetTarget("TCP:80")
+		elbHealthCheck.SetInterval(int64(30))
+		elbHealthCheck.SetTimeout(int64(5))
+		elbHealthCheck.SetHealthyThreshold(int64(2))
+		elbHealthCheck.SetUnhealthyThreshold(int64(2))
+
+		return elbHealthCheck
+	}
+
+	elbHealthCheck := &elb.HealthCheck{}
+	elbHealthCheck.SetTarget(healthCheck.Target)
+	elbHealthCheck.SetInterval(int64(healthCheck.Interval))
+	elbHealthCheck.SetTimeout(int64(healthCheck.Timeout))
+	elbHealthCheck.SetHealthyThreshold(int64(healthCheck.HealthyThreshold))
+	elbHealthCheck.SetUnhealthyThreshold(int64(healthCheck.UnhealthyThreshold))
+
+	return elbHealthCheck
+}
+
+func listenerHelper(port *models.Port) *elb.Listener {
```

Comment:
[nit] it's a little odd to have `*models.Port` as the param type when we typically work with `models.Port` objects